addons/website_sale_product_configurator/views/templates.xml

```diff
@@ -77,7 +77,7 @@
                         <t t-if="product and not combination">
                             <t t-set="combination" t-value="product_template._get_first_possible_combination()"/>
                         </t>
-                        <t t-if="not already_configured" t-call="website_sale.variants">
```

Comment:
Did you check it's not behaving strangely for people having the patched version and not updating their module?

Reply:
I just tested it right now and it looks good to me.